object_analytics_rviz/src/marker_publisher.cpp

```diff
@@ -131,11 +131,11 @@ class MarkerPublisher : public rclcpp::Node
 
       std::vector<TrackingObjectInBox> objects_tracked;
       std::vector<LocalizationObjectInBox> objects_localized;
-
       std_msgs::msg::Header header = loc->header;
       objects_tracked = tra->tracked_objects;
       objects_localized = loc->objects_in_boxes;
-
+      if (objects_tracked.size() != objects_localized.size())
```

Comment:
May I know why objects_tracked size must be the same with objects_localized size?

Reply:
can make the 3d markers displaying in RVIZ more smooth in theory.
In most cases, these two variables should be the same.